csi/main.py

```diff
@@ -42,6 +44,23 @@ def mount_storage():
     return
 
 
+def reconfigure_mounts(_signum, _frame):
+    """
+    Reconfigure the mounts by regenerating the volfiles.
+    """
+    host_volumes = get_pv_hosting_volumes({})
+    for volume in host_volumes:
+        if volume["type"] == "External":
+            # Need to skip remount external
+            continue
+        hvol = volume["name"]
+        try:
+            if reload_glusterfs(volume):
```

Comment:
Exception (cmdexception) is already handled in `reload_glusterfs` without re-raising it. I guess just `if` and `else` would suffice without exception handling?